ex5/ex5.c

```diff
@@ -16,7 +16,36 @@ char* msg3 = "hello world #3";
 
 int main(void)
 {
-    // Your code here
+    int rw[2];
+    int p = pipe(rw);
+
+    if (p < 0) {
+        printf("UH OH - PIRAHNA PLANT! PIPE FAILED.");
+        exit(1);
+    }
+
+    int child = fork();
+
+    if (child < 0) {
+        // Error
+        printf("To be, or not to be, is not the question because PIPE FAILED.");
+    } else if (child == 0) {
```

Comment:
I'm liking your comments explaining each functionality!, pseudocode is a great thing!